public/client.js

```diff
@@ -504,13 +517,15 @@ function draw() {
         for (var roomId in ROOMS) {
             var room = ROOMS[roomId];
 
+            /* for now assuming we don't care if the bg is loaded
```

Comment:
This code used to check whether things were loaded... but I think we don't need to do that anymore.